### PR TITLE
Murata IoT Device Models

### PR DESCRIPTION
**Pull request by xlinmurata**
### Thank you for contributing to the IoT Plug and Play Models repository

:memo: Please review this checklist before submission

- [ ] :eyes: The models submitted in this PR follow the [DTDL Naming Guidelines](https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines) for contributing to this repository.
- [ ] 🕵️I have validated the models locally using the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine) command line tool.
- [ ] ❗ I understand once the PR is approved, the models are considered immutable: any edits will require to submit new files.
- [ ] ✨ When creating a new top level `dtmi:namespace`, I have completed the [PR Info Template](#pr-info-template) below.

:zap: PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).

## PR Info Template

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve the _IoT Plug and Play_ ecosystem.

👇: Please replace the markdown comment examples with your own values.

### Company Info

> Info identifying your company (if applicable).

### Model Submission Goals

> Info related to broader submission goals.  

### Code Owners & Reserved Names

> Indicates GitHub alias entries to be added to the repo `CODEOWNERS` for the incoming model namespace. The codeowner is expected to be involved in subsequent DTDL model submissions against the same namespace.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

### CODE REVIEW

**Comment by xlinmurata** [comment]
Hi, 

I am trying to add the device models for my company("Murata"). It is my first project on github, I don't have any idea on what failed and why, can you please explain?
Thanks,
Xlin

**Comment by rido-min** [comment]
@xlinmurata thanks for submitting a new model.

This submission is failing because some files do not have the `@context` specifier

https://github.com/Azure/iot-plugandplay-models/actions/runs/5930729946/job/16081218156?pr=545#step:3:582

**Comment by xlinmurata** [comment]
I updated files. and now what I need to do next?

**Comment by rido-min** [comment]
> I updated files. and now what I need to do next?

commit and push to the same branch

File: dtmi/murata/demo_iot-1.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:murata:demo_iot;1",
+    "@type": "Interface",
+    "displayName": "Murata Sensors Demo Model on B-U585I-IOT02A",
+    "description": "Murata Sensors Demo Model lists interfaces, telemetry, and command options",
+    "contents": [
+        {
+            "@id": "dtmi:murata:b_u585i_iot02a:standard_fw:std_comp;3",
```
Comment (rido-min) [inline dtmi/murata/demo_iot-1.json:9]: I dont think you need this @id, but if you do it should follow the root id.

https://github.com/Azure/iot-plugandplay-models-tools/wiki/Resolution-Convention#unique-id-elements

**Review by rido-min — CHANGES_REQUESTED** [review]
the validation still fails.

You can validate locally by using dmr-client as described in https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine

**Comment by xlinmurata** [comment]
I have a change request, I have made/commit/push the change and replied with dmr-client validation result.  is there anything I need to do to trigger the PR? sorry for all the questions as I am a newbie.

**Pull request merged**